cypress/e2e/navigation.cy.ts

```diff
@@ -161,6 +169,15 @@ describe('Navigation plugin', () => {
   });
 
   describe('search and filter', () => {
+    it('should set focus to search input if plugin opened by keyboard', () => {
+      mockKalturaBe();
+      preparePage({muted: true, autoplay: true});
```

Comment:
hey @semarche-kaltura just a technical question- shouldn't {muted: true, autoplay: true} be the second argument as part of the playback config?

Reply:
Definitely should!
Good catch, fixed.